br/pkg/streamhelper/advancer.go

```diff
@@ -413,10 +413,12 @@ func (c *CheckpointAdvancer) onTaskEvent(ctx context.Context, e TaskEvent) error
 	defer c.taskMu.Unlock()
 	switch e.Type {
 	case EventAdd:
+		utils.SetLogBackupTaskExist(true)
```

Comment:
Does it work well if there are more than 1 TiDB?

Reply:
Each of tidb could watch log-backup task currently.